tests/fixtures/gtfs2ntfs/minimal/output/routes.txt

```diff
@@ -1,3 +1,3 @@
 route_id,route_name,direction_type,line_id,geometry_id,destination_id
-route:2,plop - plop,forward,route:2,,stoparea:1
-route:3,plop - plop,forward,route:3,,stoparea:1
+route:2,ma route 1,forward,route:2,,stoparea:1
```

Comment:
This modification is due to using `route_short_name` when `route_long_name` is empty (the previous implementation was leaving it empty, and therefore, the auto-generation rule was triggered).